zhaquirks/tuya/__init__.py

```diff
@@ -731,6 +731,21 @@ def handle_cluster_request(
             )
 
 
+class TZBPowerOnState(t.enum8):
+    """Tuya power on state enum."""
+
+    Off = 0x00
+    On = 0x01
+    LastState = 0x02
+
+
+class TuyaZBOnOffRestorePowerCluster(CustomCluster, OnOff):
+    """Tuya on off Zigbee cluster with restore state."""
+
+    attributes = OnOff.attributes.copy()
+    attributes.update({0x8002: ("power_on_state", TZBPowerOnState)})
```

Comment:
```suggestion
    manufacturer_attributes = {0x8002: ("power_on_state", TZBPowerOnState)}
```

Reply:
I dont have the device for testing but i have one user that have doing the same for the TS011F that can testing it on his device that have the same implantation or can i do the change without testing that is working ?